include/aspect/material_model/interface.h

```diff
@@ -359,9 +359,10 @@ namespace aspect
       template <int dim>
       void average (const AveragingOperation operation,
                     const typename DoFHandler<dim>::active_cell_iterator &cell,
-                    const Quadrature<dim>   &quadrature_formula,
-                    const Mapping<dim>      &mapping,
-                    MaterialModelOutputs    &values);
+                    const Quadrature<dim>     &quadrature_formula,
+                    const Mapping<dim>        &mapping,
+                    MaterialModelOutputs      &values_out,
+                    const MaterialModelInputs<dim>  &values_in);
     }
```

Comment:
Ok, moved it to the end, because it was originally at the end. Or do you have a preference for putting both the input and output at the beginning?